gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePlugin.java

```diff
@@ -276,6 +299,22 @@ private static List<String> checksToApplySuggestedPatchesFor(
                 .collect(Collectors.toList());
     }
 
+    private static List<String> checksToRemoveSuppressionsFor(JavaCompile javaCompile) {
+        String possibleChecksToRemove = (String) javaCompile.getProject().property(ERROR_PRONE_REMOVE_SUPPRESSIONS);
+
+        // For the suppressions to remove, if no specific check is enabled, we need to just remove everything
+        // We can't explicitly list all possible checks, because some might not exist anymore
+        // The logic itself needs to consider an empty list as "remove all"
+        if (possibleChecksToRemove == null) {
+            return List.of();
+        }
```

Comment:
Hmm, didn't think about this case. I guess it would make sense to do this? Maybe you do want to remove all automated suppressions at some point (I guess it would help with testing this plugin)

Reply:
At minimum, it makes sense to me to support it. It would also enable us to automatically open up PRs on repos to flag all existing for-rollout suppressions